bin/debug/intake_single_user.py

```diff
@@ -17,7 +17,7 @@
 
 if __name__ == '__main__':
     np.random.seed(61297777)
-
+    
```

Comment:
whitespace change, should be reverted

Reply:
I'm not sure if "git revert" would work in this case. Since the original commit removed then added a line, reverting that commit would just remove the new added line and add back the old one (which is also a whitespace change).

Reply:
But it is not a whitespace change *over the original*. Adding back to old one will keep the original whitespace, so it won't show up as a *change* and won't pollute the PR with a bunch of changes that have no real meaning.

Reply:
Oh okay! I think I get what you mean. Thanks!